presentation-compiler/src/main/dotty/tools/pc/printer/ShortenedTypePrinter.scala

```diff
@@ -351,7 +351,9 @@ class ShortenedTypePrinter(
     val paramLabelss = label(methodParams)
     val extLabelss = label(extParams)
 
-    val returnType = tpe(gtpe.finalResultType)
+    val retType = gtpe.finalResultType
+    val simplified = if (retType.typeSymbol.isAliasType) retType else retType.deepDealiasAndSimplify
+    val returnType = tpe(simplified)
```

Comment:
Won't `simplified` negatively impact union types?
```Scala
def f(): Int | String = ???
```

Reply:
Added a test, it seems to work. I don't think it will simplify something it's unable to simplify more. Do you have another example that might highlight the issue?

Reply:
It's fine then, I think it works as we'd like it to. 😄  I saw it does `lub` on `OrType` but it seems to still return an unchanged `OrType`, maybe it only collapses same cases.